File: app/src/main/java/b2w/com/br/olodjinha/ui/about/AboutFragment.java
package b2w.com.br.olodjinha.ui.about;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import b2w.com.br.olodjinha.R;
import b2w.com.br.olodjinha.MainActivity;

public class AboutFragment extends Fragment {

    public static Fragment getInstance() {
        return new AboutFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootview = inflater.inflate(R.layout.fragment_about, container, false);

        initActionBar();

        return rootview;
    }

    private void initActionBar() {
        ((MainActivity) getContext()).setAppBar(this);
    }

}

File: Lodjinha/app/src/main/kotlin/com/b2w/lodjinha/ui/about/AboutFragment.kt
package com.b2w.lodjinha.ui.about

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.b2w.lodjinha.R

class AboutFragment : Fragment(){

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? =
        inflater.inflate(R.layout.fragment_about, container, false)
}